### PR TITLE
37458 he discourse

### PR DESCRIPTION
**Pull request by zdevil**
This is part of the Discourse project. We should wait for John's approval before pushing this live.

I also added a container to the search page to make it look slightly better.

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
On hold.

**Comment by mystiberry-okta** [comment]
@zdevil or @johngronberg-okta should this be closed or is it still waiting for something?

**Comment by johngronberg-okta** [comment]
still waiting on us to complete testing for the discourse forum.

File: _source/discussion/index.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 	cta_link: "http://stackoverflow.com/search?q=okta"
 }
 show_bottom_cta: true
+redirect_to: "https://devforum.okta.com/"
 ---
 
 <section class="section--full-width text-align--center">
```
Comment (rchild-okta) [inline _source/discussion/index.html]: Nit: I'm guessing we can just remove the content in this file since we're redirecting away from the page.

**Comment by rchild-okta** [comment]
@zdevil, I see that this is still in the review stage, but one thing we'll need to do is rebase this off of `weekly` (it's the new branch we're using to gate weekly releases). Here's a rough outline:

1. Rebase your local copy against weekly

```bash
$ git fetch origin
$ git rebase -i origin/weekly
$ git push -f origin jm-OKTA-1225575-fixes-factors-url-typos:jm-OKTA-1225575-fixes-factors-url-typos
```

2. Edit the PR to merge into base 'weekly'

Note: You will have a merge conflict with footer.html (that's a file that's been recently changed in master) - I'm guessing the easiest way to resolve is to accept our changes and make the edits again.

**Comment by mystiberry-okta** [comment]
@johngronberg-okta should we close this or still waiting?

**Comment by rchild-okta** [comment]
@mystiberry-okta this got pulled into a different PR that's live now, closing.

**Pull request closed without merging**